packager/file/udp_file.cc

```diff
@@ -199,7 +199,7 @@ bool UdpFile::Open() {
 
   if (bind(new_socket.get(),
            reinterpret_cast<struct sockaddr*>(&local_sock_addr),
-           sizeof(local_sock_addr))) {
+           sizeof(local_sock_addr)) == -1) {
```

Comment:
Can you use "< 0" for consistency?

Also, just want to clarify my understanding, this is actually a clean up instead of a fix as it does not change the actual semantics. Is my understanding correct?

Reply:
> Can you use "< 0" for consistency?

Yes. I can.

> this is actually a clean up instead of a fix as it does not change the actual semantics.

Yes. Because `bind` returns `0` on success, current condition returns `false`.